tests/test_browser.py

```diff
@@ -1719,7 +1719,11 @@ def test_glgears_animation(self):
 
   @requires_graphics_hardware
   def test_fulles2_sdlproc(self):
+<<<<<<< HEAD
     self.btest('full_es2_sdlproc.c', '1', args=['-s', 'GL_TESTING', '-DHAVE_BUILTIN_SINCOS', '-s', 'FULL_ES2=1', '-lGL', '-lSDL', '-lglut'])
+=======
+    self.btest_exit('full_es2_sdlproc.c', '1', args=['-s', 'GL_TESTING=1', '-DHAVE_BUILTIN_SINCOS', '-s', 'FULL_ES2=1', '-lGL', '-lSDL', '-lglut'])
+>>>>>>> e87592c2b (tests: switch more tests to btest_exit)
```

Comment:
looks like an incomplete merge, or is github UI glitching?

Reply:
fixed